### ISSUE TITLE
Add accessibility descriptions to components by default

### ISSUE THREAD

**Issue by tasn** [callstack/react-native-paper#1676]
## Describe the feature

As the title says, components currently lack accessibility descriptions. For example, `Subheading` should have `accessibilityRole="header"` set. Many other components should have things set. At the moment everyone needs to reimplement this in their own apps, every time one is used.

Another example is `Menu.item` which is missing an appropriate role.

Heck, maybe every widget that has "onPress" set should have an appropriate role set automatically.

Another thing: TextInput accessibility is lacking too. I'd expect the `label` property to be automatically set to `accessibilityLabel` when the latter is not set.

## Motivation

Make it easier to build accessible apps.

## Related Issues

None that I could fine.

**Comment by tasn** [comment]
Still not fixed for all I can see...

**Comment by tasn** [comment]
This bot is very insistent.

**Comment by tasn** [comment]
Still broken.
